src/rosdiscover/interpreter/summary.py

```diff
@@ -90,3 +90,7 @@ def __getitem__(self, name: str) -> NodeSummary:
 
     def to_dict(self) -> List[Dict[str, Any]]:
         return [n.to_dict() for n in self.values()]
+
+    @property
+    def unresolved(self) -> Iterator[NodeSummary]:
```

Comment:
This does also seem useful, albeit orthogonal to #128

Reply:
It is used in the reporting.